types/src/front/assistant/actions/dust_app_run.ts

```diff
@@ -15,7 +15,7 @@ export type DustAppRunConfigurationType = {
 };
 
 export type DustAppParameters = {
-  [key: string]: string | number | boolean;
+  [key: string]: string | number | boolean | string[] | number[] | boolean[];
```

Comment:
Hello! That's not true in the context of DustAppRun which only supports string/number/boolean

Why do you need this?

Reply:
The idea was actually to add support for arrays in dust app parameters - as the "urls" for the [browse app](https://dust.tt/w/78bda07b39/a/21092925b9). Hence, the change in [multi-action app](https://dust.tt/w/78bda07b39/a/0e9889c787) to send the json schema to the model (with the missing "items" property), which correctly generates string[] inputs when requested to do so. The app is receiving well formed json string array.